### PR TITLE
seo bundle

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
This PR is the first one and moves the Admin stuf of CmfSeoBundle into that. For doing that the complete bundle structure needed to be created.

### CODE REVIEW

File: tests/Resources/Admin/SeoAwareContentAdmin.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        ;
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('form.group_general')
+                ->add('name', 'text')
+                ->add('title', 'text')
+                ->add('body', 'textarea', array('required' => false))
```
Comment (dbu) [inline tests/Resources/Admin/SeoAwareContentAdmin.php]: i really like wouters idea to move those to form types (that can live in the CmfSeoBundle and so on) and make the admin use those form types.

Comment (ElectricMaxxx) [inline tests/Resources/Admin/SeoAwareContentAdmin.php]: Ahh ok. Sounds valid. Got his mail after that :-) will do it.

Comment (ElectricMaxxx) [inline tests/Resources/Admin/SeoAwareContentAdmin.php]: @dbu i just realized, that the point you are pointing to is a test admin, which doesn't have any purpose on SeoBundle. SeoBundle itself has its own form type `SeoMetadataType` which still is in the SeoBundle.

Comment (dbu) [inline tests/Resources/Admin/SeoAwareContentAdmin.php]: ups, sorry.

why do we need a test admin in the bundle that provides admins? to test
the extension? i guess that makes sense then.

Comment (ElectricMaxxx) [inline tests/Resources/Admin/SeoAwareContentAdmin.php]: bingo

**Pull request merged**